prisma/schema.prisma

```diff
@@ -0,0 +1,92 @@
+// This is your Prisma schema file,
+// learn more about it in the docs: https://pris.ly/d/prisma-schema
+
+generator client {
+  provider = "prisma-client-js"
+}
+
+datasource db {
+  provider = "postgresql"
+  url      = env("DATABASE_URL")
+}
+
+enum Role {
+  ADMIN
+  USER
+}
+
+enum Visibility {
+  PUBLIC
+  PRIVATE
+  FRIENDS
+}
+
+model User {
+  id        String   @id @default(cuid())
```

Comment:
why arent we using `id      Int      @id @default(autoincrement())` for ids?

Reply:
oh you are totally right, from what i am reading, autoincrement would be faster and cuid its not need for our case